src/components/ProfileModal/ChangePasswordModal.tsx

```diff
@@ -0,0 +1,218 @@
+import { Button } from "@/components/ui/button";
+import { DialogFooter } from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
+
+import { z } from "zod";
+import cn from "classnames";
+
+import React from "react";
+import { useState, useEffect } from "react";
+
+type ChangePWProps = {
+  setProfileState: React.Dispatch<
+    React.SetStateAction<"view" | "edit" | "changePw">
+  >;
+  userData:
+    | {
+        email: string;
+        hashedPassword: string;
+        firstName: string;
+        lastName: string;
+        label: "educator" | "student" | "parent" | "administrator";
+        _id: string;
+      }
```

Comment:
This looks like it's reused in many places. Could we define it as its own type and reuse it?

**Important**, we shouldn't be sending `hashedPassword` to the frontend - it should never leave the backend for security reasons